### ISSUE TITLE
zustand seems not work well with redux devtools

### ISSUE THREAD

**Issue by blinkcat** [pmndrs/zustand#585]
As described in the [doc](https://github.com/pmndrs/zustand#redux-devtools)
```js
import { devtools } from 'zustand/middleware'

// Usage with a plain action store, it will log actions as "setState"
const useStore = create(devtools(store))
```
But when I actually tried，`Jump` to Action did't work. 
After a little debug, I found that the bug seemed to be here. `message.state` should replace the `newState`, instead of assigning it to the `DEVTOOLS ` property.
https://github.com/pmndrs/zustand/blob/d9780c78e54f08f699a36b60727968c2c3895f0f/src/middleware.ts#L126-L132
And the above seems to be related to #560, so @Aslemammad Could you please give some suggestions？:)

**Comment by dai-shi** [comment]
While waiting for @Aslemammad , a quick question: How does v3.5.10 work well?

**Comment by Aslemammad** [comment]
@dai-shi Actually, I'm waiting for a response, because i don't know what's the expected behaviour and the problem exactly! so I can solve it.

**Comment by Aslemammad** [comment]
@blinkcat Here you are! #590

**Comment by dai-shi** [comment]
closed by #590
